api/src/test/resources/protobuf-serde/messagewithany2.proto

```
@@ -0,0 +1,13 @@
+syntax = "proto3";
+package test2;
```

Buf (1.72.0)

[error] Files with package "test2" must be within a directory "test2" relative to root but were in directory "api/src/test/resources/protobuf-serde".

(PACKAGE_DIRECTORY_MATCH)